10_deploy_rook.sh

```diff
@@ -70,6 +75,11 @@ parameters:
 reclaimPolicy: Retain
 EOF
 
+# tell ceph that the rbd pool will use ~50% of the cluster
+sleep 10
```

Comment:
not sure what a better way to wait for rook to create the above pool is.. or what a safe sleep time would be.  maybe just
```
while ! ceph osd pool ls | grep rbd ; do sleep 1 ; done
```
?